lib/super_digit.rb

```diff
@@ -3,7 +3,9 @@
 # Time Complexity - ?
 # Space Complexity - ?
 def super_digit(n)
```

Comment:
👍 The time complexity is O(d^2) where d is the number of digits and it's O(d) in space complexity.